### ISSUE TITLE
Hosted API Documentation missing for Shaka Controls Events

### ISSUE THREAD

**Issue by vishwajeetv** [google/shaka-player#2504]
In the hosted API documentation, https://shaka-player-demo.appspot.com/docs/api/ 
In the events section on the bottom right, for Controls Events, if you click on it, it redirects to a 404 page, like this.
https://shaka-player-demo.appspot.com/docs/api/shaka.Controls.html#.event:TimeAndSeekRangeUpdatedEvent

Developers would have to search through the code repo via GitHub to get the exact event names.

**Comment by joeyparrish** [comment]
This sounds a lot like #2385, which we just recently fixed.  That fix has already been cherry-picked for v2.5.11, which we will be releasing soon.

I'll take a look and see if this is a separate issue, or if it's a duplicate of that.

**Comment by joeyparrish** [comment]
This is a duplicate of #2385.  It has already been fixed in the nightly build:

https://nightly-dot-shaka-player-demo.appspot.com/docs/api/shaka.ui.Controls.html#.event:LanguageSelectionUpdatedEvent

The fix will be released in v2.5.11 soon.  Hopefully this week, but we're waiting on a couple more issues to be closed.

Thanks for the report!
